src/databricks/labs/ucx/install.py

```diff
@@ -573,6 +593,14 @@ def _trigger_workflow(self, step: str):
         if self._prompts.confirm(f"Open {step} Job url that just triggered ? {job_url}"):
             webbrowser.open(job_url)
 
+    def _validate_config(self):
+        try:
+            self._ws.warehouses.get(self._config.warehouse_id)
+        except ResourceDoesNotExist:
+            logger.critical(f"warehouse does not exists anymore {self._config.warehouse_id}")
+            warehouse_id = _configure_warehouse(self._ws, self._prompts)
```

Comment:
This is a bit confusing as this is called from the `WorkspaceInstallation` where previously it was only called from the `WorkspaceInstaller`. 

Another approach would be to fetch the workspace installer like in the `AccountInstaller`, then reuse the configure warehouse from the account installer